### ISSUE TITLE
Provision alt.clas.uiowa.edu on uiowa01 as replacement for asg.clas.uiowa.edu.

### ISSUE THREAD

**Issue by mark-bennett-uiowa** [uiowa/uiowa#5604]
Provisioning alt.clas.uiowa.edu as a replacement for asg.clas.uiowa.edu due to a name change from the CLAS Dean's Office. Once this alt.clas.uiowa.edu site is provisioned, installed, and copied over from the existing asg.clas.uiowa.edu site, I will be removing the asg.clas.uiowa.edu site.

After installation, I will copy over the contents of asg.clas.uiowa.edu via the following commands (ensuring proper DB labels from Acquia):

```
drush @clasasg.prod sql:sync @self @clasalt.prod
drush @clasasg.prod rsync @self:%files @clasalt.prod:%files
```

Will create new PR for deprovision of asg.clas.uiowa.edu after alt.clas.uiowa.edu is in place and verified.

**Comment by mark-bennett-uiowa** [comment]
Migration from asg.clas.uiowa.edu to alt.clas.uiowa.edu is complete. Removing asg.clas.uiowa.edu domain and the temporary clasalt.prod.drupal.uiowa.edu domain. Once verification of success is complete, I will deprovision the asg.clas.uiowa.edu site.
